src/components/TodoCount.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { useContext } from 'react';
+import { TodosContext } from '../context/TodosContext';
+
+export const TodoCount: React.FC = () => {
+  const { todos } = useContext(TodosContext);
+  const todosLeft = todos.filter(todo => !todo.completed).length;
+
+  const countTitle = `${todosLeft} ${todosLeft === 1 ? 'items' : 'items'} left`;
```

Comment:
```suggestion
  const countTitle = `${todosLeft} ${todosLeft === 1 ? 'item' : 'items'} left`;
```

i think you wanted write like this